third_party/airflow/armada/operators/armada.py

```diff
@@ -131,3 +131,23 @@ def _get_lookout_url(self, job_id: str) -> str:
         if self.lookout_url_template is None:
             return ""
         return self.lookout_url_template.replace("<job_id>", job_id)
+
+    def on_kill(self) -> None:
```

Comment:
Can we add a unit test for this?

Reply:
try to do it

Reply:
@suprjinx , do I need to add a  new file for unit test for on_kill() operator here 'third_party/airflow/tests/unit'

Reply:
I think you could add to the existing unit test file for armada operator. I don't, however, see a lot of unit testing in this module so maybe an integration test would be easier (does the armada job show canceled state after invoking on_kill).

Reply:
This function would be called when a user cancels a dag so it would require an integration test that actually runs airflow.  We don’t actually have airflow running in our Ci at all.  Our integration test is mostly for testing interaction between armada and jobservice.

Reply:
do, I need to add this function  here [third_party/airflow/armada/operators/armada.py]? @kannon92

Reply:
Yah , thank you @kannon92